lib/standalone-single-spa.js

```diff
@@ -58,98 +58,103 @@ class StandaloneSingleSpaPlugin {
     compiler.hooks.compilation.tap(pluginName, (compilation) => {
       let publicPath;
 
-      this.options.HtmlWebpackPlugin.getHooks(
-        compilation
-      ).beforeAssetTagGeneration.tap(pluginName, (data) => {
-        publicPath = data.assets.publicPath;
-      });
+      const oldHtmlPlugin = !this.options.HtmlWebpackPlugin.getHooks;
```

Comment:
html-webpack-plugin@3 does hooks differently than v4. Compare [v3 docs](https://github.com/jantimon/html-webpack-plugin/tree/e2c6990e94b298ff66bcd885c9a03a78221479f6#events) to [v4 docs](https://github.com/jantimon/html-webpack-plugin#events)